datasets/disaster_response_messages/disaster_response_messages.py

```diff
@@ -0,0 +1,246 @@
+}
+"""
+
+_TRAIN_DOWNLOAD_URL = (
+    "https://datasets.appen.com/appen_datasets/disaster_response_data/disaster_response_messages_training.csv"
+)
+_TEST_DOWNLOAD_URL = (
+    "https://datasets.appen.com/appen_datasets/disaster_response_data/disaster_response_messages_test.csv"
+)
+
+
+class DisasterResponseMessages(datasets.GeneratorBasedBuilder):
+    """Multilingual Disaster Response Messages dataset."""
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "split": datasets.Value("string"),
+                    "message": datasets.Value("string"),
+                    "original": datasets.Value("string"),
+                    "genre": datasets.Value("string"),
+                    "related": datasets.Value("int8"),
+                    "PII": datasets.Value("int8"),
+                    "request": datasets.Value("int8"),
+                    "offer": datasets.Value("int8"),
+                    "aid_related": datasets.Value("int8"),
+                    "medical_help": datasets.Value("int8"),
+                    "medical_products": datasets.Value("int8"),
+                    "search_and_rescue": datasets.Value("int8"),
+                    "security": datasets.Value("int8"),
+                    "military": datasets.Value("int8"),
+                    "child_alone": datasets.Value("int8"),
+                    "water": datasets.Value("int8"),
+                    "food": datasets.Value("int8"),
+                    "shelter": datasets.Value("int8"),
+                    "clothing": datasets.Value("int8"),
+                    "money": datasets.Value("int8"),
+                    "missing_people": datasets.Value("int8"),
+                    "refugees": datasets.Value("int8"),
+                    "death": datasets.Value("int8"),
+                    "other_aid": datasets.Value("int8"),
+                    "infrastructure_related": datasets.Value("int8"),
+                    "transport": datasets.Value("int8"),
+                    "buildings": datasets.Value("int8"),
+                    "electricity": datasets.Value("int8"),
+                    "tools": datasets.Value("int8"),
+                    "hospitals": datasets.Value("int8"),
+                    "shops": datasets.Value("int8"),
+                    "aid_centers": datasets.Value("int8"),
+                    "other_infrastructure": datasets.Value("int8"),
+                    "weather_related": datasets.Value("int8"),
+                    "floods": datasets.Value("int8"),
+                    "storm": datasets.Value("int8"),
+                    "fire": datasets.Value("int8"),
+                    "earthquake": datasets.Value("int8"),
+                    "cold": datasets.Value("int8"),
+                    "other_weather": datasets.Value("int8"),
+                    "direct_report": datasets.Value("int8"),
```

Comment:
Can you use the `ClassLabel` feature type for those fields ?
for example
```
"fire": datasets.ClassLabel(names=["false", "true"])
```

You'll need to regenerate the dataset_infos.json file after this change